.tekton/central-db-pull-request.yaml

```
@@ -43,6 +43,10 @@ spec:
     value: ''
   - name: build-source-image
     value: 'true'
+  - name: clone-depth
+    value: '1'
```

```suggestion
  # A shallow repo clone is sufficient for central-db build.
  - name: clone-depth
    value: '1'
```
It's good to retain the original comment although it will not last for long (in subsequent tasks we may change it).